src/modules/auth.js

```diff
@@ -12,13 +12,19 @@ const auth = new JtockAuth({
   debug: false,
 });
 
-const persistLogin = async (setAuthenticated, setUid) => {
+const persistLogin = async (dispatch) => {
```

Comment:
hahahah, we had just copied from newsroom challenge.....?? 😅

Reply:
haha yes, and everyone hade reviewed it with a 'okey' ;-)